k8s-test-runner/chart/namespace_setup/setup.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: wasp-role
```

Comment:
Please remove `wasp` prefixes

Reply:
@skudasov I'll leave `wasp-role` for now as it requires changes in the infra. I can create follow up tickets to update it.